### DISCUSSION TITLE
staggered window single DIA acquisition

### DISCUSSION THREAD

**Discussion by jiyoungyoun** [vdemichev/DiaNN#252] [Q&A]
Dear Vadim, 

I have single injection DIA data acquired using 8m/z **staggered** precursor isolation window setting on Exploris 480 (following Pino L et al 2020 MCP). Is 'staggered' DIA raw files directly supported in DIA-NN? 
I'm wondering if deconvolution step required prior to inputting the 'staggered' DIA data (for instance using ProteoWizard).

Thank you,
Ji-Young

**Comment by vdemichev** [comment]
Hi Ji-Young,

Yes, staggered windows are supported by DIA-NN as is, deconvolution is not recommended.

Best,
Vadim
